src/modules/analysis/AnalysisViewComponent.js

```diff
@@ -58,6 +54,18 @@ const getTableRow = a => {
   );
 };
 
+const FieldGroup = ({ id, label, md, value }) => (
```

Comment:
cant any change. i suppose you mean the array fix. Thats done. But FieldGroup is still named FieldGroup. Might call it FormField or sumtin. Just throwing out ideas here